task1.py

```diff
@@ -0,0 +1,20 @@
+NOTATION = 16
+NOTATION_DIGITS = "0123456789ABCDEF"
+def decimal_to_other(decimal):
+    result = ""
+    
+    if decimal == 0:
+        return "0"
+    
+    while decimal > 0:
+        remainder = decimal % NOTATION
+        result = NOTATION_DIGITS[remainder] + result
```

Comment:
Если переменная используется единожды, то определять ее не обязательно.
```suggestion
        result = NOTATION_DIGITS[decimal % NOTATION] + result
```